src/main/java/bio/overture/ego/service/UserService.java

```diff
@@ -174,11 +172,16 @@ public User getUserByToken(@NonNull IDToken idToken) {
         findByProviderTypeAndProviderId(providerType, providerId)
             .or(
                 () -> {
```

Comment:
This is a little hard to read. Refactor this into a method that is called in the `.or` statement.

Reply:
This comment is for the current functional component passed to the `.or()` statement